p2p/utils_test.go

```diff
@@ -0,0 +1,263 @@
+package p2p
+
+import (
+	"crypto/ed25519"
+	"encoding/binary"
+	"sort"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	typesP2P "github.com/pokt-network/pocket/p2p/types"
+	mocksP2P "github.com/pokt-network/pocket/p2p/types/mocks"
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/runtime/test_artifacts"
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/modules"
+	modulesMock "github.com/pokt-network/pocket/shared/modules/mocks"
+	telemetry "github.com/pokt-network/pocket/telemetry"
+	"github.com/stretchr/testify/require"
+)
+
+// ~~~~~~ RainTree Unit Test Configurations ~~~~~~
+
+// TODO: We should leverage the shared `test_artifacts` package and replace the code below.
```

Comment:
We are currently trying to remove the test_artifacts package and fully encapsulate tests to their respective modules. I don't think this `TODO` is appropriate